packages/server/tests/acceptance/rpc_batch1.spec.ts

```diff
@@ -61,6 +61,21 @@ describe('@api-batch-1 RPC Server Acceptance Tests', function () {
   const ONE_TINYBAR = Utils.add0xPrefix(Utils.toHex(Constants.TINYBAR_TO_WEIBAR_COEF));
   const sendRawTransaction = relay.sendRawTransaction;
 
+  const resolveEvmAddress = async (tx: any) => {
```

Comment:
Add comments on the function to explain behaviour and function signature

Reply:
Pushed in the fix